cadc-pkg-server/src/main/java/org/opencadc/pkg/server/PackageRunner.java

```diff
@@ -206,7 +210,7 @@ private void doIt() {
         } catch (Throwable t) {
             if (ThrowableUtil.isACause(t, InterruptedException.class)) {
                 try {
-                    ep = jobUpdater.setPhase(job.getID(), ExecutionPhase.QUEUED, ExecutionPhase.EXECUTING, new Date());
+                    ep = jobUpdater.setPhase(job.getID(), getInitialPhase(), ExecutionPhase.EXECUTING, new Date());
```

Comment:
this is something that was like this and I never noticed before but it's really broken:

this change from initial to executing in the error handling doesn't make any sense

should be EXECUTING to ERROR with an error message from the throwable and should always call sendError (I don't know what scenario that ABORTED is meant to handle)

out of scope for now: there is also no code here at differentiate the cause and chose an approp status code - that would normally be done by earlier `catch` blocks and this is the final "unexpected" aka 500 handling... please create an issue for this

Reply:
Shouldn't the catch for the throwable just be setting the phase to ERROR and calling sendError(), without checking if the throwable cause is a particular exception?

Reply:
yes, as long as the fail is not from doing initial->executing you can be certain that the current phase is executing and go to error.

and because this is really  `catch(Throwable unexpected)` the response code should be 500 in this block.

Reply:
in the throwable you can get the current job phase when setting the phase to error, which would cover the phase in initial or executing?